content/news/2021-08-19-gsoc-tools.md

```diff
@@ -0,0 +1,61 @@
++++
+title = "Devin's Google Summer of Code Ending"
+date = 2021-08-19
+extra.author = "Devin Ragotzy"
++++
+
+# A Summer of Ruma Tooling Comes to an End
+
+What a fun summer! I was hoping, this year I would become more a part of the Rust community, and
+I think I have. I learned how to open an MCP (Major Change Proposal), which is how Rust makes
```

Comment:
Aren't MCPs about the implementation (compiler internals) rather than the language as a whole?

Reply:
Well, there's `compiler` MCP and `lang` MCP so maybe I should be more clear? A few people told me to do one and then I was told to do the other :laughing: so not really sure which was right.

Reply:
Ohh right there's that distinction.

Reply:
Maybe sth. like "which is a process that language and implementation changes not big enough for an RFC go through"?